users.md

```diff
@@ -10,6 +10,7 @@ aim to remove the differences and provide a
 *   in Java:
     https://github.com/bazelbuild/bazel/tree/master/src/main/java/net/starlark/java
 *   in Rust: https://github.com/google/starlark-rust/
+*   in C and Lua: https://github.com/obazl/moonlark
```

Comment:
It's not an implementation of the language, just a tool.
It can be added below, although I'm not certain if it can be recommended (vs bazelbuild/buildtools).

Reply:
Good point, it's a parser-only project (no interpreter). Moved to the tool section.